pkg/blobstore/configuration/create_blob_access.go

```diff
@@ -261,12 +261,41 @@ func createBlobAccess(configuration *pb.BlobAccessConfiguration, options *blobAc
 				backend.Redis.ReplicationCount,
 				replicationTimeout)
 		case *pb.RedisBlobAccessConfiguration_Single:
+
+			dialTimeout := 5 * time.Second
```

Comment:
Is there any need to provide explicit default values? What about just declaring them like this?
```
var dialTimeout time.Time
```
That way we can simply defer to go-redis for the default values.

Reply:
There was no reason in the end, I had it exactly as you have pointed out at the start and I was going to set my own default values which I then didn't do so I will change it back.